src/api/inventory/search.ts

```diff
@@ -0,0 +1,70 @@
+import { search } from 'src/api/search/search';
+
+const getSearchPayload = (speciesId: number | string, organizationId: number | string, count: number) => {
```

Comment:
this explicit search fields is intentional - as it is a static search for summaries.
we can refactor this once all other inventory search is built out - such as for seedlings batches etc.